readme.md

```diff
@@ -1,12 +1,9 @@
-[![CircleCI](https://circleci.com/gh/forcedotcom/SalesforceMobileSDK-iOS/tree/dev.svg?style=svg)](https://circleci.com/gh/forcedotcom/SalesforceMobileSDK-iOS/tree/dev)
```

Comment:
Why are we removing `CircleCI` status?

Reply:
1) On master, we shouldn't show dev status.
2) The majority of the time, the build are broken - but not because of the code.

Reply:
- Our test runs only run against `dev`. Having status against `master` wouldn't really help because no changes go into `master` typically. We decided to show status for `dev` because most open source projects show status of latest builds. `master` is the default branch, so anyone that lands on the repo will be shown `master`, which should show the latest status of nightly builds.
- We should fix the tests so that the status shown here is green.